sass/components/CreateDiskRow/create-disk-row.scss

```diff
@@ -0,0 +1,3 @@
+.kubevirt-create-disk-row__action-buttons {
```

Comment:
You are correct, I've missed that.

Since we need to target specific buttons, we can use `.kubevirt-create-disk-row__action-button` (singular) class name on `<Button>` components, if you agree.

We don't have to declare `.kubevirt-create-disk-row__action-buttons` (plural) if we don't need to style the enclosing `<div>` (button panel/container).

> `.kubevirt-create-disk-row__action-buttons button`
> `.kubevirt-create-disk-row__action-buttons .btn`

Nested CSS selectors should only be used when we don't have full control over the logical component's parts. In this case, we can simply use `.kubevirt-create-disk-row__action-button`, since PF-React's `<Button>` allows us to specify CSS class name.

`.block tag` selectors should be avoided if possible. CSS rules shouldn't be tied to HTML tags.

`.block .nested-part` selectors are possible, but unnecessary in this case.